package.json

```diff
@@ -487,8 +497,10 @@
         "@types/node": "^16.18.13",
         "@types/sinon": "^10.0.13",
         "@types/vscode": "1.69.0",
+        "@types/xml2js": "^0.4.14",
         "@typescript-eslint/eslint-plugin": "^5.60.1",
         "@typescript-eslint/parser": "^5.60.1",
+        "@vscode/dts": "^0.4.0",
```

Comment:
It's used to download vscode.d.ts and vscode.proposed.d.ts, can remove when releasing stable